website/docs/docs/core/connect-data-platform/redshift-setup.md

```diff
@@ -46,10 +46,59 @@ pip is the easiest way to install the adapter:
 
 <p>For further info, refer to the GitHub repository: <a href={`https://github.com/${frontMatter.meta.github_repo}`}>{frontMatter.meta.github_repo}</a>.</p>
 
+## Configurations
 
-## Authentication Methods
+| Profile field | Example | Description |
+| ------------- | ------- | ------------ |
+| `type` | redshift | The type of data warehouse you are connecting to|
+| `host` | hostname.region.redshift.amazonaws.com| Host of cluster |
+| `port`  | 5439 |   |
+| `dbname`  | my_db | Database name|
+| `schema`  | my_schema | Schema name| 
+| `connect_timeout`  | `None` or 30 | Number of seconds before connection times out| 
+| `sslmode`  | prefer | optional, set the sslmode to connect to the database. Default prefer, which will use 'verify-ca' to connect. For more information on `sslmode`, see Redshift note below| 
+| `role`  | None | Optional| 
+| `autocreate`  | false | Optional, default false. Creates user if they do not exist | 
+| `db_groups`  | ['ANALYSTS'] | Optional. A list of existing database group names that the DbUser joins for the current session | 
+| `ra3_node`  | true | Optional, default False. Enables cross-database sources| 
+| `autocommit`  | true | Optional, default True. Enables autocommit after each statement| 
+| `retries`  | 1 | Number of retries | 
+| `region`  | us-east-1 | Optional | 
```

Comment:
perhaps we only mention this parameter within the IAM auth section?
```suggestion
```

Reply:
Good call!